spec/unit/util/ldap/connection_spec.rb

```diff
@@ -158,4 +158,28 @@ class SSLConn < Conn; end
       Puppet::Util::Ldap::Connection.instance
     end
   end
+  
+  describe "when creating a connection with ldapcrtdir" do
+    it "should support specifing a ldapcrtdir" do
+      lambda { Puppet::Util::Ldap::Connection.new("myhost", "myport", :crtdir => "/var/ldap") }.should_not raise_error
```

Comment:
change to `expect(...).to_not raise_error` or just execute `Connection.new` and the test will fail if it raises automatically.